aiohttp/web.py

```diff
@@ -133,6 +135,8 @@ def _set_loop(self, loop):
         if self._debug is ...:
             self._debug = loop.get_debug()
 
+        self._jobs = JobRunner(loop=loop)
```

Comment:
May be stupid idea, but I think we may want to pass JobRunner as parameter. I suspect with time, we will see interesting custom implementation of runner, for instance `ExecutorJobRunner`, that uses `Executor`